plugins/lang/common.txt

```diff
@@ -538,9 +538,9 @@ YES = Sim
 NO = Não
 BAN = banir
 KICK = expulsar
-NO_ACC_COM = Tu nao tens acesso a esse comando
+NO_ACC_COM = Tu não tens acesso a esse comando
 USAGE = Uso
-MORE_CL_MATCHT = Tem mais do que um cliente que corresponde ao teu argumento
+MORE_CL_MATCHT = Há mais do que um cliente a corresponder o teu argumento
 CL_NOT_FOUND = O cliente com esse nome ou userid não foi encontrado
 CLIENT_IMM = O cliente "%s" tem imunidade
 CANT_PERF_DEAD = Essa ação não pode ser executada no cliente morto "%s"
```

Comment:
Wouldn't it be "num cliente morto"?

Reply:
No, "num cliente morto" translates to "on **a** dead client". The english translation is "on dead client "%s"", so it's "no cliente morto "%""

Reply:
Absolutely, my bad

Reply:
No problem.